src/signals/incident-management/containers/DefaultTextsAdmin/components/DefaultTextsForm/index.js

```diff
@@ -15,11 +15,20 @@ import { ChevronDown, ChevronUp } from '@datapunt/asc-assets';
 
 const StyledWrapper = styled.div`
   margin-top: 33px;
-  width: 100%;
+  flex-basis: 100%;
```

Comment:
Only makes sense having this rule if `display: flex;` has been set too

Reply:
`the parent is a <Column> so flex-basis  is correct here`

Reply:
Yep, you're right.